src/ef-args.c

```diff
@@ -6,6 +6,12 @@
 #include <stdio.h>
 
 int argc_frame(int argc, const char *argv[], frame_t *f) {
+    
+    if(argc > FRAME_STACK_MAX) {
```

Comment:
Can you move this check after the variables are declared on the stack.

Reply:
Sure.